### DISCUSSION TITLE
Custom dictionaries -- CSV file location?

### DISCUSSION THREAD

**Discussion by bakerds** [akvorado/akvorado#1048] [Q&A] [answered]
When running Akvorado in the suggested Docker environment, where does it look for the CSV file used by custom dictionaries?

https://demo.akvorado.net/docs/configuration#custom-dictionaries

**Comment by vincentbernat** [comment, marked as answer]
You can put them in the configuration directory and refer to them as `/etc/akvorado/something.csv`.

**Reply by bakerds** [reply]
I was about to submit a docs PR, and I see you have already made it more clear. Thanks!
